<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogCommandSwitchTexture</class>
 <widget class="QDialog" name="DialogCommandSwitchTexture">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>612</width>
    <height>223</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>15</number>
   </property>
   <property name="topMargin">
    <number>10</number>
   </property>
   <property name="rightMargin">
    <number>15</number>
   </property>
   <property name="bottomMargin">
    <number>10</number>
   </property>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0">
     <property name="spacing">
      <number>15</number>
     </property>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QCheckBox" name="checkBoxTilesetID">
         <property name="text">
          <string>Tileset ID:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="PanelPrimitiveValue" name="panelPrimitiveTilesetID" native="true">
         <property name="minimumSize">
          <size>
           <width>250</width>
           <height>27</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>250</width>
           <height>27</height>
          </size>
         </property>
         <property name="acceptDrops">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="WidgetPicture" name="widgetPictureTileset" native="true">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>27</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>27</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QCheckBox" name="checkBoxAutotileID">
         <property name="text">
          <string>Autotile ID:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="PanelPrimitiveValue" name="panelPrimitiveAutotileID" native="true">
         <property name="minimumSize">
          <size>
           <width>250</width>
           <height>27</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>250</width>
           <height>27</height>
          </size>
         </property>
         <property name="acceptDrops">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="WidgetPicture" name="widgetPictureAutotile" native="true">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>27</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>27</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QCheckBox" name="checkBoxWallID">
         <property name="text">
          <string>Wall ID:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="PanelPrimitiveValue" name="panelPrimitiveWallID" native="true">
         <property name="minimumSize">
          <size>
           <width>250</width>
           <height>27</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>250</width>
           <height>27</height>
          </size>
         </property>
         <property name="acceptDrops">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="WidgetPicture" name="widgetPictureWall" native="true">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>27</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>27</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QCheckBox" name="checkBox3DObjectID">
         <property name="text">
          <string>3D object ID:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="PanelPrimitiveValue" name="panelPrimitive3DObjectID" native="true">
         <property name="minimumSize">
          <size>
           <width>250</width>
           <height>27</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>250</width>
           <height>27</height>
          </size>
         </property>
         <property name="acceptDrops">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="WidgetPicture" name="widgetPicture3DObject" native="true">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>27</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>27</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QCheckBox" name="checkBoxMountainID">
         <property name="text">
          <string>Mountain ID:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="PanelPrimitiveValue" name="panelPrimitiveMountainID" native="true">
         <property name="minimumSize">
          <size>
           <width>250</width>
           <height>27</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>250</width>
           <height>27</height>
          </size>
         </property>
         <property name="acceptDrops">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="WidgetPicture" name="widgetPictureMountain" native="true">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>27</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>27</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PanelPrimitiveValue</class>
   <extends>QWidget</extends>
   <header>panelprimitivevalue.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WidgetPicture</class>
   <extends>QWidget</extends>
   <header>widgetpicture.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogCommandSwitchTexture</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogCommandSwitchTexture</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
